plugins/GSdx/GSSetting.cpp

```diff
@@ -46,6 +46,7 @@ const char* dialog_message(int ID, bool* updateText) {
 #endif
 		case IDC_CRC_LEVEL:
 			return "Control the number of Auto-CRC hacks applied to games.\n\n"
+				"Automatic:\nAutomatically select 'Partial' or 'Full' level based on the selected renderer.\n\n"
```

Comment:
Well, I like the vagueness. The string doesn't need to be updated as much that way. How about:

> Automatically select the recommended CRC hack level based on the selected renderer. Recommended setting.
Partial will be chosen for OpenGL.
Full will be chosen for Direct3D.